backend/src/main/java/com/fellas/bespoke/converter/AnnotationCreateRequestToASRequest.java

```diff
@@ -14,23 +14,23 @@
     private static final String BODY_TYPE = "TextualBody";
     private static final String TYPE = "Annotation";
     private static final String SELECTOR = "CssSelector";
-    private static final String REFINED_BY_TYPE = "TextQuoteSelector";
-    private final TimeZone TIME_ZONE = TimeZone.getTimeZone("UTC");
-    private final DateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+    private static final String REFINEDBYTYPE = "TextQuoteSelector";
+    private final TimeZone TIMEZONE = TimeZone.getTimeZone("UTC");
+    private final DateFormat DATEFORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
```

Comment:
Dont use upper case notations unless variable static. In this case itShould be dateFormat or it should be static as well.